sklearn/preprocessing/tests/test_discretization.py

```diff
@@ -51,7 +62,9 @@
 )
 def test_fit_transform(strategy, expected, sample_weight):
```

Comment:
Let's update this test to test (and maybe others) to test for the `quantile_method="averaged_inverted_cdf"` whenever strategy is `"quantile"` instead: this will become the default so better test the future default whenever we test for hardcoded values in our tests.